docs/workflow.rst

```diff
@@ -2,30 +2,22 @@
 Overall work flow and software architecture
 *******************************************
 
-Figure 1 shows a flowchart that captures the overall functionality of
-viroconcom. A statistical model of the offshore environment can be created
-by fitting a model structure to measured data. Then, this statistical model
-can be used to construct an environmental contour.
+Figure 1 shows a flowchart that captures the overall functionality of virocon. A statistical model of the (offshore)
+environment can be created by fitting a model structure to measured data. Then, this statistical model can be used to
+construct an environmental contour.
 
-.. figure:: viroconcom-flowchart.jpg
-    :scale: 50 %
-    :alt: flowchart for viroconcom
-
-    Figure 1. Flowchart showing how the process of fitting a model structure to
-    measured data and constructing an environmental contour.
+.. figure:: virocon_flowchart.png

```

Comment:
I feel like the items in the figure could be shortened a little so that there are at max 3. lines. 
E.g. Define structure of joint distribution model -> Define structure of joint model
Estimate the model's parameter values (fitting) -> Estimate the model parameters

I think we can sacrifice a little "correctness" for the sake of readability.